.github/workflows/cicd.yaml

```diff
@@ -176,7 +178,6 @@ jobs:
           POSTGRES_PORT: 5432
           PGUSER: username
           PGPASSWORD: password
-          PGHOST: localhost
```

Comment:
Why is this removed here (and in the docker yaml files)?

Reply:
https://github.com/stac-utils/pgstac/issues/166#issuecomment-1446525911 (we don't know why it was needed but it is)